packs/spell-effects/spell-effect-competitive-edge.json

```diff
@@ -80,34 +57,13 @@
             },
             {
                 "key": "FlatModifier",
-                "label": "Competitive Edge (Enhanced Attack)",
```

Comment:
This one should be done as an adjust modifier. Removing the labels would leave both modifiers with the same slug.

Reply:
I changed it to an AdjustStrike and tested it.

Reply:
Looks good